include/rocksdb/db.h

```diff
@@ -179,10 +179,35 @@ class DB {
                                     const std::string& column_family_name,
                                     ColumnFamilyHandle** handle);
 
+  // Bulk create column families the same column family options.
+  // Return the handles of the column families through the argument handles.
+  // In case of error, the request may success partially, and handles will
+  // contain column family handles that it managed to create.
```

Comment:
Consider saying what size `handles` will have in this case: will it be resized to `column_family_names.size()` but contain some nullptr's, or will it be shorter?